src/main/java/org/asamk/signal/dbus/DbusManagerImpl.java

```diff
@@ -907,6 +908,12 @@ private List<MessageEnvelope.Data.Attachment> getAttachments(final Map<String, V
         }).toList();
     }
 
+    @Override
+    public File getAttachmentFile(final AttachmentPointer pointer) {
+        //TODO may need an implementation
+        return null;
```

Comment:
Just throw an exception for now, as the dbus interface doesn't support retrieving attachments yet.

```suggestion
        throw new UnsupportedOperationException();
```

Reply:
I changed that